source/help/getting-started/creating-an-account.rst

```diff
@@ -0,0 +1,53 @@
+=================================================
+Creating an account and signing in the first time
+=================================================
+
+It's a relatively straight forward process to create an account on Mattermost and start communicating with other people. You enter some basic information on the account creation page then click **Create Account**.
+
+.. Important::
+  - The account creation process might be different than described here, depending on the policies of your organization.
+  - If your administrator sent you instructions, use those instructions.
+  - If you are not able to create an account using these instructions, contact your administrator.
+
+On the account creation page, you must enter the following information:
+
+email address
+  Enter the address that you want to use for signing in, receiving notifications, resetting passwords, and receiving your account confirmation email.
+user name
+  Enter the text that you want displayed in Mattermost. This is the name that people will know you by. If enabled by your administrator, you can also use this name to sign in.
+password
+  Enter a password that is at least 5 characters long. Your administrator might configure more restrictive requirements, such as the use of numbers, upper and lower case, and special characters. If the password that you enter does not meet one or more of the minimum requirements, you see a message that tells you what they are. You can then modify your password to meet the requirements.
+
+To create an account
```

Comment:
Propose this section just becomes the intro to the doc rather than its own section? Seems out of order with the workflow of how a person would sign up.